scripts/check_ui_online.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+
+kibana_syslog=/var/log/syslog
+
+grep -m 1 "bundles compiled successfully" <(tail -f -n +1 $kibana_syslog)
```

Comment:
it's cumbersome to detect Kibana finished building, this works a least in recent minors, but in case it doesn't work, deployment needs to be checked manually